### PR TITLE
feat!: use updated outbound OpenAPI specification from api-snippets

### PR DESCRIPTION
**Pull request by kirgene**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -566,85 +553,6 @@ jobs:
           event: fail
           template: SLACK_TEMP_RELEASE_FAILURE
 
-  publish-npm:
```
Comment (mdebarros) [inline .circleci/config.yml:569]: Why was this removed?

**Review by mdebarros — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -37,19 +37,6 @@ defaults_license_scanner: &defaults_license_scanner
     git clone https://github.com/mojaloop/license-scanner /tmp/license-scanner
     cd /tmp/license-scanner && make build default-files set-up
 
-## remove
-defaults_npm_auth: &defaults_npm_auth
```
Comment (mdebarros) [inline .circleci/config.yml:41]: Please undo.

File: .circleci/config.yml [hunk 3]
```diff
@@ -37,19 +37,6 @@ defaults_license_scanner: &defaults_license_scanner
...
-  name: Update NPM registry auth token
-  command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
-
-## remove
-defaults_npm_publish_release: &defaults_npm_publish_release
```
Comment (mdebarros) [inline .circleci/config.yml:46]: Please undo.

File: .circleci/config.yml [hunk 4]
```diff
@@ -860,41 +768,3 @@ workflows:
             branches:
               ignore:
                 - /.*/
-      - publish-npm:
```
Comment (mdebarros) [inline .circleci/config.yml:863]: Please undo.

File: .circleci/config.yml [hunk 5]
```diff
@@ -860,41 +768,3 @@ workflows:
...
-            - license-scan
-            - image-scan
-            - validate-openapi
-          filters:
-            tags:
-              only: /v[0-9]+(\.[0-9]+)*/
-            branches:
-              ignore:
-                - /.*/
-      - publish-npm-snapshot:
```
Comment (mdebarros) [inline .circleci/config.yml:882]: Please undo.

File: package.json [hunk 6]
```diff
@@ -81,24 +79,24 @@
...
-    "ws": "^8.8.0"
+    "ws": "^8.8.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.18.6",
-    "@babel/preset-env": "^7.18.6",
-    "@mojaloop/api-snippets": "^14.0.0",
+    "@babel/core": "^7.18.9",
+    "@babel/preset-env": "^7.18.9",
+    "@mojaloop/api-snippets": "^14.2.0",
```
[the remark this answers is not in the document]
Comment (mdebarros) [reply package.json:87]: I believe it does now since the Interface is now part of the API Snippet --> https://github.com/mojaloop/sdk-scheme-adapter/pull/340/files#diff-8edf0cb22838239441d348ba38b95b757052d8834f21c1ee61b07a9a77174106R93

**Review by mdebarros — APPROVED** [review]
+1

**Review by mdebarros — CHANGES_REQUESTED** [review]
CI checks are failing?

File: package.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (mdebarros) [reply package.json:87]: Along with this, I have deprecated the [package/@mojaloop/sdk-scheme-adapter](https://www.npmjs.com/package/@mojaloop/sdk-scheme-adapter), with the following message provided by @kleyow.

```
The use case of this library was to support Typescript types for the sdk-scheme-adapter's outbound interface for integration purposes. This functionality is now superseded by @mojaloop/api-snippets which now hosts the outbound interface. api-snippets generates and exports these types and can be imported using "import { SDKSchemeAdapter } from '@mojaloop/api-snippets'"
```

[the remark this answers is not in the document]
Comment (kirgene) [reply package.json:87]: Yes, it's a requirement now:
https://github.com/mojaloop/sdk-scheme-adapter/blob/85207fc5c265f73cc3e33075e3019c5561130b5f/src/OutboundServer/index.js#L93

[the remark this answers is not in the document]
Comment (mdebarros) [reply package.json]: I do have a concern here about moving the Interfaces from the SDK into the api-snippets, specifically around the “discoverability” of the Interface for anybody who is interested in using the Outbound API.

@kirgene please make sure you update the README to clearly include links to both Inbound and Outbound interfaces for consistency.

[the remark this answers is not in the document]
Comment (kirgene) [reply package.json]: @mdebarros have updated README

**Review by kleyow — APPROVED** [review]

**Review by mdebarros — APPROVED** [review]
+1
